gulpfile.js

```diff
@@ -121,7 +122,7 @@ gulp.task('compile-tests', function () {
 
 gulp.task('default', function () {
     runSequence(
-        'watch',
-        $.shell.task('PATH=node_modules/.bin:$PATH electron .')
+        'clean',
+        ['typescript', 'sass', 'react']
```

Comment:
You're right. I think for now I will do a simple check of platform, if the platform is darwin then the shell.task will be added. This until I found a better way :smile: